scripts/gceworker.sh

```diff
@@ -67,14 +60,15 @@ case "${cmd}" in
     fi
 
     # Install automatic shutdown after ten minutes of operation without a
-    # logged in user. To disable this, `sudo touch /.active`.
+    # logged in user. To disable this for 2 days, `sudo touch /.active`.
     gcloud compute ssh "${NAME}" --command="sudo cp bootstrap/autoshutdown.cron.sh /root/; echo '* * * * * /root/autoshutdown.cron.sh 10' | sudo crontab -i -"
+    gcloud compute ssh "${NAME}" --command="sudo reboot"
```

Comment:
?

Reply:
It's better to reboot after the initial bootstrap, so the machine picks up the kernel updates we get after `apt dist-upgrade.